src/main/scala/com/buransky/plugins/scoverage/util/PathUtil.scala

```diff
@@ -28,9 +28,10 @@ import scala.Iterator
  * @author Rado Buransky
  */
 object PathUtil {
+  type PathSeq = Vector[String]
```

Comment:
Any particular reason for making this a `Vector` as opposed to keeping it as an abstract `Seq`?

Reply:
Ok, that's a fair point.